### ISSUE TITLE
[🐛BUG] Possible Bug wih Model NCEPLRec

### ISSUE THREAD

**Issue by PosoSAgapo** [RUCAIBox/RecBole#1539]
Hi,
I am training the NCEPLRec on my own dataset with my own user, item, inter file. The following is my config:

```
#model config
rank: 25
beta: 1.0
reg_weight: 1e2
#dataset config
data_path: 'recbole_format_dataset/'
filed_separator: "\t"
seq_separator: " "
USER_ID_FIELD: user
ITEM_ID_FIELD: item
NEG_PREFIX: neg_
device: cpu
load_col:
  inter: [user, item]
  user: [user, zip_code]
  item: [item, feat_A, feat_B, feat_C, feat_D]
epochs: 500
train_batch_size: 256
learner: adam
learning_rate: 0.0005
eval_step: 1
stopping_step: 30
train_neg_sample_args: {distribution: 'uniform', sample_num: 10}
# evaluation setting
eval_setting: R0_RS, full
group_by_uesr: True
split_ratio: [0.9, 0.1, 0]
metrics: ["Recall", "MRR", "NDCG", "Hit", "Precision"]
topk: [50]
valid_metric: Hit@50
eval_batch_size: 256
```
Then my code:
```
config = Config(model='NCEPLRec', dataset='hkust', config_file_list=['NCEPLRec.yaml'])
init_seed(config['seed'], config['reproducibility'])
train_data, valid_data, test_data = data_preparation(config, dataset)
model = NCEPLRec(config, train_data.dataset).to(config['device'])
trainer = Trainer(config, model)
best_valid_score, best_valid_result = trainer.fit(train_data,valid_data, saved=True, show_progress=config["show_progress"])

```
The console outputs: 
1. The trainable parameter size is only 1
2. The loss on the train set converges to 0.000 only after 1 epoch.
I have adjusted the rank parameter several times from 25 to 450, but the behavior is still the same. 
Is above problem the model feature or bug?

**Comment by AoiDragon** [comment]
@PosoSAgapo Hello, 

NCE-PLRec is not a common deep learning method for recommendation. It's a matrix factorization method with no trainable parameters and hence no loss function. Since RecBole requests every model to have at least one parameter, we set parameter size to 1. Also, The training loss is set to 0.

**Comment by PosoSAgapo** [comment]
@AoiDragon 
Thanks for your detailed explanation. If the training loss is set to 0, does this mean the validation performance will not change during the training time? I notice that the validation performance is constant after the first epoch.

**Comment by AoiDragon** [comment]
Yes, you're right.
